swift/integration-tests/posix-only/symlinks/test.py

```diff
@@ -0,0 +1,20 @@
+from create_database_utils import *
+import os
```

Comment:
```suggestion
from pathlib import Path
```
or my earlier proposed change won't work